.github/workflows/deploy-frontend-azurepreview.yml

```
@@ -477,6 +483,134 @@ jobs:
+            --name $APP \
+            --resource-group ${{ env.RG }} \
+            --query properties.configuration.ingress.fqdn -o tsv)
+
+          echo "url=https://$URL" >> $GITHUB_OUTPUT
+
+  nexus-pr-comment:
+    needs: [check, nexus]
+    if: needs.check.outputs.run_nexus == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
```

actionlint (1.7.12)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)